net/local/local_netpoll.c

```diff
@@ -119,7 +119,10 @@ static void local_inout_poll_cb(FAR struct pollfd *fds)
 {
   FAR struct pollfd *originfds = fds->arg;
 
-  poll_notify(&originfds, 1, fds->revents);
+  if (fds->revents != 0)
```

Comment:
should we move the check of revent to poll_notify?

Reply:
Move this check out of poll notify to make epoll_default_cb can know which fd is notifed. As mentioned in the commit, some drivers' poll implementations need we call the poll_setup() again when their internal states changed (e.g., local socket), so I move this check to poll callback and  epoll can know which fd is notified and setup it again in the next epoll_wait call.

Reply:
can we add a flag(e.g. POLLALWAYS)?